subsystem/cluster_test.go

```diff
@@ -218,6 +219,22 @@ var _ = Describe("Cluster", func() {
 		Expect(c.Hosts[0].ID.String()).Should(Equal(h.ID.String()))
 	})
 
+	It("Deregister host triggers cluster validations", func() {
```

Comment:
Please also add a test for a negative case, where `refreshClusterStatus` returns an error.

Reply:
It is not an interesting case since the update process just fails. What important here was to verify that the validations are returned synchronously.

Reply:
I disagree.
Unit tests are meant to assert possible outcomes.
We should assert that if cluster sync fails, we return an error here.

Reply:
Good point on both accounts. Adding a unit test